src/client/js/Dialogs/PluginConfig/PluginConfigDialog.js

```diff
@@ -74,8 +76,21 @@ define(['js/Controls/PropertyGrid/PropertyGridWidgetManager',
         this._divContainer = this._dialog.find('.modal-body');
 
         this._saveConfigurationCb = this._dialog.find('.save-configuration');
+        this._modalHeader = this._dialog.find('.modal-header');
 
-        this._title = this._dialog.find('.modal-title');
+        if (typeof this._pluginMetadata.icon === 'string') {
```

Comment:
What would be the order/precedence? Would we apply the iconClass on the img tag, too?